### PR TITLE
Implement live-metrics skeleton

### PR DESCRIPTION
**Pull request by lzchen**
Adding live metrics feature skeleton. This PR is just the skeleton which includes dummy methods. Majority of this PR is including the swagger definition and auto-generation of the required classes.

### CODE REVIEW

File: sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## 1.0.0b21 (2024-01-16)
 
+### Features Added
+
+- Add live metrics skeleton + swagger definitions
+    ([#33983](https://github.com/Azure/azure-sdk-for-python/pull/33983))
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md:8]: Move to unreleased version

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/__init__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+from azure.monitor.opentelemetry.exporter.quickpulse._exporter import QuickPulseExporter
+
+__all__ = [
+    "QuickpulseExporter",
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/__init__.py:10]: QuickP*ulse

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/__init__.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# -------------------------------------------------------------------------
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/__init__.py:1]: Thought we may eventually open it to customers, wondering if we should privatize this module for now.

File: sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/_exporter.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    Histogram: AggregationTemporality.DELTA,
+    ObservableCounter: AggregationTemporality.DELTA,
+    ObservableGauge: AggregationTemporality.CUMULATIVE,
+    ObservableUpDownCounter: AggregationTemporality.CUMULATIVE,
+    UpDownCounter: AggregationTemporality.CUMULATIVE,
+}
+
+
+class QuickPulseExporter(MetricExporter):
+    """Azure Monitor Metric exporter for OpenTelemetry."""
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/quickpulse/_exporter.py:40]: nit: Docstring needs updating

File: sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry Exporter for Azure Monitor Client for Python
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md:1]: Change title for specifically quick pulse

File: sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (jeremydvoss) [inline sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md:2]: nit: why not include in the same readme?

Comment (lzchen) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md:2]: Taking a look at the command for autorest: `autorest ./swagger/README_QUICKPULSE.md --python --v3`, it takes a single README file and i dont believe you can define multiple `yamls` in the same README.

Comment (jeremydvoss) [reply sdk/monitor/azure-monitor-opentelemetry-exporter/swagger/README_QUICKPULSE.md:2]: Oh didn't realize it was functional too. Alright

**Review by jeremydvoss — APPROVED** [review]

**Pull request merged**